### PR TITLE
Added the main Tagline

### PR DESCRIPTION
**Pull request by shahajishinde**
Made the site's main title short by dividing it into two lines. Also added a short tagline which would require everyone's concern. Made small changes to my previous about us section. Nothing much of significance done.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,9 +67,10 @@
     <header>
         <div class="header-content">
             <div class="header-content-inner">
-                <h1 id="homeHeading">RIT India Linux Users' Group</h1>
+                <h1 id="homeHeading">RIT INDIA</h1>
+                <h1 id="homeHeading">LINUX USERS' GROUP</h1>
```
Comment (bhavin192) [inline index.html]: Any specific reason to have all in capital?

Comment (bhavin192) [inline index.html:71]: Any specific reason to have all in capital?

File: index.html [hunk 2]
```diff
@@ -67,9 +67,10 @@
...
                 <hr>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas facilisis odio non neque fringilla, id varius purus consectetur. Nam fermentum tortor id nisl commodo</p>
+                <p>"Where Learning is Fun and Sharing is a Learning methodology."</p>
```
Comment (bhavin192) [inline index.html:73]: LGTM

File: index.html (hunk 1 above)
Comment (akshaygin) [reply index.html:71]: No problem with capital, all the ```h1``` tag under ```header-content-inner``` transform into uppercase

**Comment by akshaygin** [comment]
Good job. 👍 Thanks @shaharao

**Pull request merged**